AGENTS.md

```
@@ -0,0 +1,189 @@
+
+## Tech Stack
+
+- `api/`: Go 1.16 + Echo v4 + GORM v1.25 + PostgreSQL
+- `mobile/lib/`: Flutter（Dart >= 3.6.0）、`fvm` 管理、`Hive` + `SharedPreferences` で永続化
+- `gas/`: Google Apps Script（スプレッドシートにバインド）
+
+`admin/` と `raw/` は使用していません。
+
+## Commands
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)